app/login/page.tsx

```diff
@@ -70,6 +72,14 @@ const LoginPage: React.FC = () => {
 
     const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
         event.preventDefault();
+        setLoading(true);
+        setErrors([]); // Clear previous errors
+
+        if (attempts >= 5) {
+            setErrors(['Too many attempts. Please wait before trying again.']);
```

Comment:
If we are setting it to 15 minutes, we should be specific in the message

Reply:
@briri do you think 15 minutes makes sense to you? It was just a random number.

Reply:
@briri I updated the code to use a more dynamic approach to lockout times, making it more difficult for bad actors to commit brute force attacks. I also included the remaining minutes in the message to the user.

Reply:
Yeah, I think 15 makes sense. Thanks for making it dynamic :)